packages/use-growthbook/src/AbTestProvider.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error TODO: remove once Growthbook is correctly typed and export
+import { GrowthBook, GrowthBookProvider } from '@growthbook/growthbook-react'
+import type { ReactNode } from 'react'
+import { useCallback, useEffect, useMemo } from 'react'
+import type { GrowthBookType } from './types'
+
+export type ToolConfig = {
```

Comment:
Can we pick type from Growthbook here ?

Reply:
You mean official typing from growthbook or do you mean moving this type in the types.ts file ?